testcases/OpTestEM.py

```diff
@@ -514,32 +516,36 @@ def runTest(self):
             before_time[i] = self.c.run_command("cat /sys/devices/system/cpu/cpu*/cpuidle/state%s/time" % (i))
             before_time[i] = [int(a) for a in before_time[i]]
 
+        # Enable one idle state, check residency, disable and repeat.
         after_usage = {}
         after_time = {}
         for i in idle_states:
+            success = 0
+            total = 0
             self.enable_idle_state(i)
             self.verify_enable_idle_state(i)
-            for c in range(nrcpus):
-                self.c.run_command("taskset 0x%x find / |head -n 200000 > /dev/null" % (1 << c))
+            cpus = []
+            cmd = """for cpu in {0..%d};do taskset -c $cpu sha1sum temptext.txt & done; wait""" % (nrcpus -1)
```

Comment:
In petitboot, tasklet is busybox version. It doesn't have the option c. This will fail when we run the testcase `testcases.OpTestEM.cpu_idle_states_skiroot` .
```
[console-expect]#for cpu in {0..127};do taskset -c $cpu sha1sum temptext.txt & done; wait
taskset: invalid option -- 'c'
BusyBox v1.27.2 (2018-05-31 10:41:48 PDT) multi-call binary.

Usage: taskset [-p] [HEXMASK] PID | PROG ARGS

Set or get CPU affinity

	-p	Operate on an existing PID
[1]+  Done(1)                    taskset -c ${cpu} sha1sum temptext.txt
```